components/bl-list-component/src/list-content.js

```diff
@@ -0,0 +1,53 @@
+import { List } from './list';
+
+export function ListContent({
+  component, eventHandlers, child}) {
+
+  const { color, fontSize, listItems, type} = component;
+
+  const itemStyles = {
+    color   : color,
+    fontSize: fontSize
+  };
+
+  const onItemClick = (key) => {
+    eventHandlers.onClickListItem({item: key});
+  }
+
+ const getItems = (items) => {
+    return (items || []).map((item, i) => {
+      if (item.child && item.child.length !== 0) {
```

Comment:
this cons
```suggestion
    return (items || []).map((item, i) => {
      if (item.child && item.child.length !== 0) {
```

this construction should hone after refactoring to List and ListItem component.